packages/saved-views-react/src/api/utilities/translation/SavedViewTranslation.ts

```diff
@@ -122,7 +99,7 @@ async function translateSavedViewToLegacySavedView(
     legacySavedView = savedViewData.legacyView as any;
     // legacySavedView.id = savedView.id; // Change legacy sv id to comboId
 
-  } else if (isSavedViewItwin3d(savedViewData)) {
+  } else if (isViewDataITwin3d(savedViewData)) {
```

Comment:
Why SavedView -> View? Is there other types of views

Reply:
The method name is translateSavedViewToLegacySavedView. Should this be translateViewToLegacyView?

Reply:
> Why SavedView -> View? Is there other types of views

There's a type union `ViewData` defined as `ViewDataITwin3d | ViewDataITwinDrawing | ViewDataITwinSheet`.

> The method name is translateSavedViewToLegacySavedView. Should this be translateViewToLegacyView?

It is not clear without knowing the full context what "legacy" means. At some point we will need to go through all uses of this word and replace it with something that is more accurate.